docs/content/documentation/templates/overview.md

```diff
@@ -251,6 +251,15 @@ template:
 }
 ```
 
+The `bibtex` format loads data into a structure matching the format used by the
+[nom-bibtex crate](https://crates.io/crates/nom-bibtex). The following example prints the author
```

Comment:
Can this be detailed in the doc itself? The README on crates.io has no information and I don't see where it is described on the docs either. https://docs.rs/nom-bibtex/0.3.0/nom_bibtex/enum.Entry.html ?

Reply:
I'm not sure why the README.md doesn't show up in crates.io. It does show up on [github](https://github.com/charlesvdv/nom-bibtex) and in main page of [docs.rs](https://docs.rs/nom-bibtex/0.3.0/nom_bibtex/).

In any case I agree that there should be more details in the documentation, I'll have a jab at it and report back. Thanks!

Reply:
I've just pushed an update to the documentation, please have look.